src/content/guides/build-performance.md

```diff
@@ -14,7 +14,7 @@ This guide contains some useful tips for improving build/compilation performance
 
 ## General
 
-The following best practices should help whether or not you are in [development](/guides/development) or building for [production](/guides/production).
+The following best practices should help, whether you're building in [development](/guides/development) or [production](/guides/production).
```

Comment:
Don't want to seem too picky but perhaps this can be further reworded to: `building` -> `running build scripts`?

Reply:
Yeah! I like that.